docs/en/sql-reference/data-types/lowcardinality.md

```diff
@@ -0,0 +1,57 @@
+---
+toc_priority: 51
+toc_title: LowCardinality
+---
+
+# LowCardinality data type {#lowcardinality-data-type}
+
+Modifies other data types.
+
+## Syntax {#lowcardinality-syntax}
+
+```sql
+LowCardinality(data_type)
+```
+
+**Parameters**
+
+- `data_type` — Supported ClickHouse [data type](index.md).
+
+## Description {#lowcardinality-dscr}
+
+`LowCardinality` is a superstructure that changes a data storage method and rules of data processing. ClickHouse applies [dictionary coding](https://en.wikipedia.org/wiki/Dictionary_coder) to `LowCardinality`-columns. Operating with dictionary encoded data significantly increases performance of [SELECT](../statements/select/index.md) queries for many applications.
+
+Consider using `LowCardinality` instead of [Enum](enum.md) when working with strings. `LowCardinality` provides more flexibility in use and often reveals the same or higher efficiency.
```

Comment:
needs some guidelines on how to choose String vs LowCardinality(String)

Reply:
Most exanations are in See Also articles. I think we don't need to repeat it here.

Reply:
I think it worth mentioning.
It should be faster, in general for < 10k distinct values, and may be slower for > 100k distinct values. But it is better to test for each case anyway.